README.md

```diff
@@ -25,34 +25,33 @@ MSBuild 15.9 builds from the branch [`vs15.9`](https://github.com/Microsoft/msbu
 |**.NET Core**|[![Build Status](https://ci2.dot.net/buildStatus/icon?job=Microsoft_msbuild/vs15.9/innerloop_Windows_NT_CoreCLR)](https://ci2.dot.net/job/Microsoft_msbuild/job/vs15.9/job/innerloop_Windows_NT_CoreCLR)|[![Build Status](https://ci2.dot.net/buildStatus/icon?job=Microsoft_msbuild/vs15.9/innerloop_Ubuntu16.04_CoreCLR)](https://ci2.dot.net/job/Microsoft_msbuild/job/vs15.9/job/innerloop_Ubuntu16.04_CoreCLR)|[![Build Status](https://ci2.dot.net/buildStatus/icon?job=Microsoft_msbuild/vs15.9/innerloop_OSX10.13_CoreCLR)](https://ci2.dot.net/job/Microsoft_msbuild/job/vs15.9/job/innerloop_OSX10.13_CoreCLR)|
 
-### Source code
-
-* Clone the sources: `git clone https://github.com/Microsoft/msbuild.git`
-
 ## Building
 
-### Building MSBuild with Visual Studio 2019
+### Building MSBuild with Visual Studio 2019 on Windows
 
 For the full supported experience, you will need to have Visual Studio 2019 or higher.
 
 To get started on **Visual Studio 2019**:
 
 1. [Install Visual Studio 2019](https://www.visualstudio.com/vs/).  Select the following Workloads:
-  - _.NET desktop development_
-  - _.NET Core cross-platform development_
-2. [Install the .NET Core 2.1 SDK](https://www.microsoft.com/net/learn/get-started/windows).
-2. Clone the source code (see above).
-2. Open a `Developer Command Prompt for VS 2019` prompt.
-3. Build the code using the `build.cmd` script at the root of the repo. This also restores packages needed to open the projects in Visual Studio.
-5. Open `MSBuild.sln` or `MSBuild.Dev.sln` in Visual Studio 2019.
+  - .NET desktop development
+  - .NET Core cross-platform development
+2. [Install the .NET Core SDK](https://dotnet.microsoft.com/download).
+3. Open a `Developer Command Prompt for VS 2019` prompt.
+4. Clone the source code: `git clone https://github.com/Microsoft/msbuild.git`
+  - You may have to [download git](https://git-scm.com/downloads) first.
+5. Run `.\build.cmd` from the root of the repo to build the code. This also restores packages needed to open the projects in Visual Studio.
+6. Open `MSBuild.sln` or `MSBuild.Dev.sln` in Visual Studio 2019.
+
+Note: To create a usable MSBuild with your changes, run `.\build.cmd /p:CreateBootstrap=true`.
 
-Note: To produce a "bootstrap" build, run `.\build.cmd /p:CreateBootstrap=true`.
+This newly-built MSBuild will be located at `artifacts\bin\bootstrap\net472\MSBuild\Current\Bin\MSBuild.exe`. It may not work for all scenarios, including C++ builds.
 
 ### Building MSBuild in Unix (Mac & Linux)
 
 MSBuild can be run on Unix systems that support .NET Core. Set-up instructions can be viewed on the wiki: [Building Testing and Debugging on .Net Core MSBuild](documentation/wiki/Building-Testing-and-Debugging-on-.Net-Core-MSBuild.md)
 
-## Localization
+## Localized Builds
```

Comment:
Why this change? Since the paragraph includes a link to "how to contribute a translation change" I like the more generic name.